SQL/SQL (basic).sublime-syntax

```diff
@@ -0,0 +1,221 @@
+%YAML 1.2
+---
+name: SQL (Basic)
+scope: source.sql.basic
+hidden: true
+version: 2
+
+variables:
+  string_escape: (?:\\.)
+  dml_targets: (?:aggregate|conversion|database|domain|function|group|((?:fulltext|spatial|unique)\s+)?index|language|operator class|operator|procedure|rule|schema|sequence|table(?:space)?|trigger|type|user|view)
+
+contexts:
+  prototype:
+    - include: comments
+
+  main:
+    - match: |-
+        (?xi)
+        \b(create(?:\s+or\s+replace)?)\s+
+        ({{dml_targets}})
+        \b\s*
```

Comment:
Probably it makes sense to have a separate context for this instead of (or as well as) a variable, to improve the highlighting while typing experience.